examples/box_two_modes.jl

```diff
@@ -0,0 +1,73 @@
+"Box model with two modes"
```

Comment:
Specify in title and docstring that they are gamma modes; maybe call it box_gamma_mixture ?